packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2732,10 +2742,6 @@ export class ContainerRuntime
 			return;
 		}
 
-		if (!canSendOps) {
-			this.documentsSchemaController.onDisconnect();
-		}
```

Comment:
@Vladsud - Can you confirm that this call was only needed in the case where we had a pending Document Schema op which failed to make it to the ordering service?  i.e. the resubmit case?  I moved this call (and renamed it) to resubmit, just wanted to make sure I don't also need it here for some reason I can't see yet.

cc @scottn12 and @ChumpChief as other people familiar with Document Schema

Reply:
I had to read the code to refresh my memory, but yes, your assumption looks correct.